romfs/source/fighter/mario/param/vl.prcxml

```diff
@@ -5,6 +5,19 @@
       <float hash="p1_y">21.5</float>
     </struct>
   </list>
+  <list hash="param_fireball">
+    <struct index="0">
+      <float hash= "speed">1.5</float>
+      <float hash= "gravity_accel">0.08</float>
+      <float hash= "angle">-10</float>
+      <float hash= "bound_ref_speed_mul">0.92</float>
+      <float hash="bounded_speed_y_min">1.5</float>
+      <float hash="bounded_speed_y_max">2.5</float>
+    </struct>
+    <hash40 index="1">dummy</hash40>
+    <hash40 index="2">dummy</hash40>
+    <hash40 index="3">dummy</hash40>
+  </list>
```

Comment:
Please include a changelog describing the specific change in param values from vanilla to this PR

Reply:
> 

Got it